src/components/Content/index.tsx

```diff
@@ -143,6 +78,67 @@ export const Content: React.VFC = () => {
   );
 };
 
+export const Content: React.VFC = () => (
```

Comment:
Should we separate these components into different files? They seem fairly distinct 😳

Reply:
done and I will be careful about including the changes from your PR in case that goes in first 🫡